lib/cisco_node_utils/vrf_af.rb

```diff
@@ -142,6 +182,21 @@ def default_route_target_export_evpn
     end
 
     # --------------------------
+    def route_target_export_stitching
+      cmds = config_get('vrf', 'route_target_export_stitching', @get_args)
+      cmds.nil? ? nil : cmds.sort
+    end
```

Comment:
Discussed with @chrisvanheuveln and I understand the design for this property and the route_target_export_evpn property better now. This should be fine.